modules/installing-ocp-agent-ibm-z.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/installing-with-agent-based-installer.adoc
```

Comment:
Yes correct, since IBM Z currently only works when through PXE assets I think it better fits into the PXE [assembly](https://github.com/openshift/openshift-docs/pull/69850/files#diff-f54c6c49df16118f8aa38f436a4b946609d0c77741c6bb37835417d583c435a4).